RoboticMaterials/rmstudio
# Add reset password to authentication page

To test:

1. Create an account
2. Click change password
3. Make sure you get an email with the password reset code
4. Create a new password
5. Make sure the new password works for logging in
6. Sign out from app
